packages/o/openplayerjs.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "openplayerjs",
+  "filename": "openplayer.min.js",
+  "description": "Lightweight HTML5 video/audio player with smooth controls and ability to play VAST/VPAID/VMAP ads.",
+  "homepage": "https://www.openplayerjs.com/",
+  "keywords": [
+    "media",
+    "video",
+    "audio",
+    "hls",
+    "dash",
+    "mp4",
+    "mp3",
+    "openplayer",
+    "openplayerjs",
+    "vast",
+    "vpaid",
+    "player",
+    "video-player",
+    "audio-player",
+    "html5-media",
+    "hlsjs",
+    "ima-sdk"
+  ],
+  "repository":{
+    "url": "https://github.com/openplayerjs/openplayerjs.git",
+    "type": "git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/openplayerjs/openplayerjs.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": ["**/*"]
+      },
+      {
+        "basePath": "types",
+        "files": ["**/*"]
```

```suggestion
        "basePath": "",
        "files": [
          "types/**/*.d.ts"
        ]
```
Let's keep `types` in their subdirectory and restrict the wildcard so that it's not open and dangerous.